annotations/models.py

```diff
@@ -288,7 +288,10 @@ def sort_key(self):
         return sort_key(sentence.xml_id, sentence.XML_TAG)
 
     def first_sentence(self):
-        return self.sentence_set.first()
```

Comment:
`.first()` seems to do the same thing as the new code, right?

Reply:
the difference is that `first()` triggers a new DB query, while `[0]` uses prefetched data

Reply:
Cool :+1: I suspected as much when I was looking at the other changes :-) Could you please add that as a comment?